srv/src/main/java/my/bookshop/handlers/SubscriptionHandler.java

```diff
@@ -0,0 +1,30 @@
+package my.bookshop.handlers;
+import org.springframework.stereotype.Component;
+
+import com.sap.cds.services.handler.EventHandler;
+import com.sap.cds.services.handler.annotations.Before;
+import com.sap.cds.services.handler.annotations.ServiceName;
+import com.sap.cds.services.mt.MtAsyncUnsubscribeEventContext;
+import com.sap.cds.services.mt.MtSubscriptionService;
+import com.sap.cds.services.mt.MtUnsubscribeEventContext;
+
+	/**
+	 * Handler that implements subscription logic
+	 */
+	@Component
+	@ServiceName(MtSubscriptionService.DEFAULT_NAME)
+	public class SubscriptionHandler implements EventHandler {
```

Comment:
The class part has one indentation, which should be removed. Also we could think about enabling this handler only when the `cloud` profile is active, to prevent the error when running locally that no `MtSubscriptionService` could be found for registration of the event handler.